lib/ci/tar-fetcher/watch.rb

```diff
@@ -149,6 +146,18 @@ def uscan(chdir, destdir)
       )
     end
 
+    def uscan_lenient(chdir, destdir)
```

Comment:
Not sure I like this new download type being a copy of the old type. Couldn't we just read the debian/watch and detect at runtime whether which pile of arguments we need and then construct a suitable cmdline?